docs/react.md

```diff
@@ -120,21 +120,12 @@ const history = channel.history((err, result) => {
 });
 ```
 
-It's also worth highlighting that the `useChannel` hook supports all of the additional parameters that the regular Ably SDK does as we're simply passing the call along. See the [Ably Channel docs](https://ably.com/docs/channels) for more info.
+We support providing [ChannelOptions](https://ably.com/docs/api/realtime-sdk/types#channel-options) to the `useChannel` hook:
 
 This means you can use features like `rewind`:
 
 ```javascript
-const { channel } = useChannel("[?rewind=100]your-channel-name", (message) => {
-    // This call will rewind 100 messages
-    console.log(message);
-});
-```
-
-We support providing [ChannelOptions](https://ably.com/docs/api/realtime-sdk/types#channel-options) to the `useChannel` hook:
-
-```javascript
-const { channel } = useChannel({ channelName: "your-channel-name", options: { ... } }, (message) => {
+const { channel } = useChannel({ channelName: "your-channel-name", options: {params: {rewind: '1'}}}, (message) => {
```

Comment:
Minor thing but I thing it will be consistent to code style:
```suggestion
const { channel } = useChannel({ channelName: "your-channel-name", options: { params: { rewind: '1' } } }, (message) => {
```